systemtest/src/main/java/io/strimzi/systemtest/resources/keycloak/SetupKeycloak.java

```diff
@@ -127,10 +128,10 @@ private static KeycloakInstance createKeycloakInstance(String namespaceName) {
         Secret keycloakSecret = kubeClient().getSecret(namespaceName, KEYCLOAK_SECRET_NAME);
 
         String usernameEncoded = keycloakSecret.getData().get("username");
-        String username = new String(Base64.getDecoder().decode(usernameEncoded.getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
+        String username = new String(Util.decodeBytesFromBase64(usernameEncoded.getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
```

Comment:
Here, but also in the other cases of this ... should these go directly to String instead using the already existing `Utils.decodeFromBase64(String data, Charset charset)` method? I think when the `byte[]` is actually used, `Util.decodeBytesFromBase64` makes sense. but where we just create a String from it right away, we should use the method to decode it directly to String, or?

Reply:
Yes, thats a good point, I've updated the implementation